service/src/main/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcDocumentTaskExecutor.java

```diff
@@ -0,0 +1,19 @@
+package uk.nhs.adaptors.gp2gp.gpc;
+
+import lombok.extern.slf4j.Slf4j;
+import uk.nhs.adaptors.gp2gp.common.task.TaskDefinition;
+import uk.nhs.adaptors.gp2gp.common.task.TaskExecutor;
+
+@Slf4j
+public class GetGpcDocumentTaskExecutor implements TaskExecutor {
```

Comment:
Once generic type is implemented in interface, this will be declared as

```
public class GetGpcDocumentTaskExecutor implements TaskExecutor<GetGpcDocumentTaskDefinition>
```